public/js/Home.js

```diff
@@ -0,0 +1,53 @@
+const requirements = document.getElementById('requirements');
+const textArea = document.getElementById('text-area');
+const skills = document.getElementById('skills');
+const addMore = document.getElementById('add-more');
+const deletBtn = document.getElementById('delete');
+const post = document.getElementById('post');
+const i = 0;
+let j = 0;
+
+function append() {
+  requirements.style.display = 'block';
+}
+addMore.addEventListener('click', () => {
+  j++;
+  const newSkill = document.createElement('input');
+  newSkill.setAttribute('type', 'text');
+  newSkill.setAttribute('placeholder', 'Enter a required skill.');
+  newSkill.setAttribute('id', j);
```

Comment:
@Anies12 I wonder if you could clone a skill input element that's already on the page.  Then you would get all of the attributes (text, placeholder, any class that you add in the future) set for you automatically.  You'd still have to update the id.  Something like this -
```
const newSkill = skills.querySelector('input').cloneNode(true);
newSkill.setAttribute('id', j);
...
```